packages/ontario-design-system-component-library/src/components/ontario-task/ontario-task.scss

```diff
@@ -0,0 +1,57 @@
+@use 'sass:math';
+@use '@ongov/ontario-design-system-global-styles/dist/styles/scss/1-variables/spacing.variables' as spacing;
+@use '@ongov/ontario-design-system-global-styles/dist/styles/scss/1-variables/colours.variables' as colours;
+@use '@ongov/ontario-design-system-global-styles/dist/styles/scss/2-tools/functions/global.functions' as globalFunctions;
+@use '@ongov/ontario-design-system-global-styles/dist/styles/scss/1-variables/breakpoints.variables' as breakpoints;
+
+.ontario-task {
+	border-bottom: 2px solid colours.$ontario-greyscale-20;
```

Comment:
Replace hardcoded with appropriate variables. I am not sure if these exist but it's just an example.
```
border-bottom: spacing.$border-width-thick solid colours.$ontario-greyscale-20;
padding: spacing.$spacing-4 spacing.$spacing-0;
```